### PR TITLE
support both ee-feature pack and preview feature pack #48

### PR DESCRIPTION
**Pull request by liweinan**

### CODE REVIEW

File: resteasy-spring-feature-pack/galleon-feature-pack/pom.xml [hunk 1]
```diff
@@ -23,90 +23,28 @@
...
         <groupId>org.jboss.resteasy.spring</groupId>
-        <artifactId>resteasy-spring-parent</artifactId>
+        <artifactId>resteasy-spring-feature-pack</artifactId>
         <version>3.0.0-SNAPSHOT</version>
     </parent>
     <modelVersion>4.0.0</modelVersion>
 
     <artifactId>galleon-feature-pack</artifactId>
-    <name>RESTEasy Spring: Galleon Feature Pack</name>
+    <name>RESEasy Spring: Galleon Feature Pack</name>
```
Comment (jamezp) [inline resteasy-spring-feature-pack/galleon-feature-pack/pom.xml:32]: This lost the "T" in "RESTEasy"

**Review by jamezp — COMMENTED** [review]
This looks like it makes sense. I converted it to a draft because it can't be merged with a SNAPSHOT dependency IMO.

File: resteasy-spring-feature-pack/galleon-feature-pack/pom.xml (hunk 1 above)
Comment (liweinan) [reply resteasy-spring-feature-pack/galleon-feature-pack/pom.xml:32]: yes sir!

**Comment by liweinan** [comment]
This PR needs RESTEasy 6.2.1.Final to merge

**Review by jamezp — APPROVED** [review]
Looks good.

**Pull request merged**